src/components/questions/questions-list/QuestionGroupInfo.js

```diff
@@ -19,18 +19,15 @@ const MemberInfoWrapper = styled.div`
 `
 
 const QuestionGroupInfo = ({ questionGroup }) => {
-  const memberQuery = useMemberNameAndContractMarketInfoQuery({
-    variables: { memberId: questionGroup.memberId },
-  })
 
-  const memberDataMaybe = memberQuery?.data?.member
+  const memberDataMaybe = questionGroup?.member
   return (
     <QuestionGroupInfoWrapper memberId={questionGroup.memberId}>
       <MemberInfoWrapper>
         {memberDataMaybe ? (
           <>
             {memberDataMaybe?.firstName} {memberDataMaybe?.lastName}{' '}
-            {getMemberFlag(memberDataMaybe?.contractMarketInfo?.market)},{' '}
+            {getMemberFlag(memberDataMaybe?.contractMarketInfo?.market)}
           </>
         ) : null}
```

Comment:
Maybe I'm missing something and there's a good reason to why, but could we do some better naming here? 🧐
And also skip `: null` and instead go for something like:

```suggestion
  const member = questionGroup?.member
  return (
    <QuestionGroupInfoWrapper memberId={questionGroup.memberId}>
      <MemberInfoWrapper>
        {member && 
          <>
            {member.firstName} {member.lastName}{' '}
            {getMemberFlag(member.contractMarketInfo?.market)},{' '}
            {getMemberFlag(member.contractMarketInfo?.market)}
          </>
          }
```

Reply:
I agree, && is better suited here. Nice spot!